WebClient/src/main/resources/webapp-server.yml

```diff
@@ -0,0 +1,11 @@
+# Spring Properties
+spring:
+  application:
+     name: web-app  # Service registers under this name
+  freemarker:
+    enabled: false      # Ignore Eureka dashboard FreeMarker templates
+  thymeleaf:
+    cache: false        # Allow Thymeleaf templates to be reloaded at runtime
+    prefix: classpath:/static/
+server:
+  port: 1234   # HTTP (Tomcat) port
```

Comment:
Oh! There are 2 servers! Nice of you to differentiate them so clearly! 1223 is OBVIOUSLY much different from 1234. How could one POSSIBLY mistype!